app/models/user.rb

```diff
@@ -12,5 +14,33 @@ class User < ActiveRecord::Base
   devise :database_authenticatable, :registerable,
     :recoverable, :rememberable,
     :trackable, :validatable,
-    :omniauthable
+    :omniauthable, :omniauth_providers => [:linkedin]
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.